scripts/zkai_attention_kv_fused_softmax_table_section_delta_gate.py

```
@@ -0,0 +1,887 @@
+            "proof_size_bytes",
+            "envelope_size_bytes",
+            "section_bytes",
+            "section_payload_bytes_total",
+            "json_wrapper_bytes",
+            "proof_backend",
+            "proof_backend_version",
+            "statement_version",
+            "proof_schema_version",
+        }
+        if set(artifact) != expected_artifact:
+            raise FusedSoftmaxTableSectionDeltaGateError("artifact profile field drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)